Assets/Sanji.cs

```diff
@@ -13,6 +14,8 @@ public class Sanji : Character {
 
 		player.knockbackVector = new Vector2 (direction > 0 ? -1 : 1, 1);
 		player.wasThrown (direction);
+
+		hitter.GetComponent<Health> ().Damage (kickDmg);
```

Comment:
Another thought I had here, how about making the Health component a variable in PlayerMovement in the same way it has HitBoxManager?
```
// in PlayerMovement.cs
public Health health;
void Start() {
  ...
  health = GetComponent<Health>();
}

// Then this line in Sanji.cs would be
player.health.Damage(kickDmg);
```
Not necessary though, just a thought.